src/NLog.Targets.MauiLog/NLog.Targets.MauiLog.csproj

```diff
@@ -1,72 +1,126 @@
+		
+		<VersionPrefix>5.2.1</VersionPrefix>
+		<AssemblyVersion>5.0.0.0</AssemblyVersion>
+		<AppVeyorBuildNumber>$(APPVEYOR_BUILD_NUMBER)</AppVeyorBuildNumber>
+		<AppVeyorBuildNumber Condition="'$(AppVeyorBuildNumber)' == ''">0</AppVeyorBuildNumber>
+		<FileVersion>$(VersionPrefix).$(AppVeyorBuildNumber)</FileVersion>
+		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+
+		<Title>NLog.Targets.MauiLog</Title>
+		<Company>NLog</Company>
+		<Description>NLog MauiLog Target for writing to Apple Unified Logging / Android.Util.Log for LogCat</Description>
+		<RepositoryType>git</RepositoryType>
+		<RepositoryUrl>https://github.com/NLog/NLog.Targets.MauiLog.git</RepositoryUrl>
+		<PackageTags>Maui;Xamarin;Forms;Android;MonoDroid;MacOS;iOS;LogCat;OSLog;NSLog;NLog;logging;log</PackageTags>
+		<PackageProjectUrl>https://github.com/NLog/NLog.Targets.MauiLog</PackageProjectUrl>
+		<PackageIcon>N.png</PackageIcon>
+		<PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<PackageReleaseNotes>
+			- Added support for UnhandledExceptionEventHandler in RegisterMauiLog
+			- Updated to NLog v5.2.2
+			- Added nuget-package README.md
+
+			See https://github.com/NLog/NLog.Targets.MauiLog for documentation of NLog targets for MAUI
+		</PackageReleaseNotes>
+
+		<IsPackable>true</IsPackable>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<EnableTrimAnalyzer>true</EnableTrimAnalyzer>
+		<IsTrimmable>true</IsTrimmable>
+		<CheckEolWorkloads>false</CheckEolWorkloads>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="NLog" Version="5.2.2" />
+		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="ILLink.Descriptors.xml">
+			<LogicalName>ILLink.Descriptors.xml</LogicalName>
+		</EmbeddedResource>
+	</ItemGroup>
+
+	<ItemGroup>
+		<None Include="../../README.md" Pack="true" PackagePath="" />
+		<None Include="N.png" Pack="true" PackagePath="" Visible="false" />
+	</ItemGroup>
+	
+	<Target Name="DownloadMissingContent" BeforeTargets="GenerateNuspec">
+		<DownloadFile SourceUrl="https://nlog-project.org/N.png" DestinationFolder="$(MSBuildThisFileDirectory)" />
+	</Target>
+
+	<PropertyGroup Condition="$(TargetFramework.StartsWith('net7.0-android')) == true">
+		<DefineConstants>$(DefineConstants);__ANDROID__</DefineConstants>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="$(TargetFramework.StartsWith('net7.0-ios')) == true">
+		<DefineConstants>$(DefineConstants);__APPLE__</DefineConstants>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="$(TargetFramework.StartsWith('net7.0-maccatalyst')) == true">
+		<DefineConstants>$(DefineConstants);__APPLE__</DefineConstants>
+	</PropertyGroup>
+
+	<!-- Android -->
+	<ItemGroup Condition="$(TargetFramework.StartsWith('net7.0-android')) != true">
```

Comment:
Instead of using `$(TargetFramework.StartsWith('net7.0-android')) != true` could it become `$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'android'` ?